.github/workflows/build.yml

```diff
@@ -69,6 +81,29 @@ jobs:
             exit 1
           fi
 
+      - name: Build MX Util
+        run: |
+          cd $GITHUB_WORKSPACE/mx-tools
+          # In future Will use make file to build
+          docker build ./src --build-arg MX_VERSION=${{env.MX_VERSION}} -t mendixlabs/mx-docker-tools:${{env.MX_VERSION}}
+
+      - name: Run MX Util
+        run: |
+          MX_TOOLS_IMAGE="mendixlabs/mx-docker-tools:"${{env.MX_VERSION}}
+          mkdir $GITHUB_WORKSPACE/app/Source/packages
+          docker run -v $GITHUB_WORKSPACE/app/Source:/opt/app -i $MX_TOOLS_IMAGE mxutil.exe create-module-package --package-dir /opt/app/packages /opt/app/ExpenseRequestStarterApp.mpr "WorkflowCommons"
+          cp -f $GITHUB_WORKSPACE/app/Source/packages/WorkflowCommons.mpk $GITHUB_WORKSPACE/app/Releases/WorkflowCommon-Latest.mpk
+          rm -rf $GITHUB_WORKSPACE/app/Source/packages
+
+      - name: Commit release
```

Comment:
Wouldn't it create an infinite loop of `create package`, `commit`, `triggers build`, `create package`, `commit`, `triggers build`? Maybe it would make sense to make it conditional?

Reply:
Based on two answers https://stackoverflow.com/questions/57921401/push-to-origin-from-github-action and https://github.community/t/workflow-infinite-loop/16547 I added this line
`git remote set-url origin https://x-access-token:${{ secrets.GITHUB_TOKEN }}@github.com/${{ github.repository }}`